TASVideos.WikiEngine/NodeImplementations/Module.cs

```diff
@@ -5,12 +5,24 @@ public class Module : INode
 	public NodeType Type => NodeType.Module;
 	public string Name { get; }
 	public IReadOnlyDictionary<string, string> Parameters { get; }
-	public int CharStart { get; }
-	public int CharEnd { get; set; }
-	public Module(int charStart, int charEnd, string text)
+
+	private StringIndices _charRange = (default, default);
```

Comment:
This field init was necessary at one point, but in the end only the tuple ctor is left, so it can be removed.